screen-orientation/src/definitions.ts

```diff
@@ -1,9 +1,22 @@
 import type { PluginListenerHandle } from '@capacitor/core';
 
 export interface OrientationLockOptions {
+  /**
+   * Note: OrientationLockType was removed from typescript >= 5.2.
```

Comment:
The note is not very clear, I think it should mention that typescript 5.2 users might need to import the `OrientationLockType` from the plugin.

Reply:
I've rewritten this note